docs/guides/extension_points.rst

```diff
@@ -142,6 +145,8 @@ Here are the different integration points that python plugins can use:
 .. _learning_context.py: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/xblock/learning_context/learning_context.py
 .. |UserPartition docstring| replace:: ``UserPartition`` docstring
 .. _UserPartition docstring: https://github.com/edx/edx-platform/blob/f8cc58618a39c9f7b8e9e1001eb2d7a10395797e/common/lib/xmodule/xmodule/partitions/partitions.py#L105-L120
+.. |pluggable_override docstring| replace:: ``pluggable_override`` docstring
+.. _pluggable_override docstring: https://github.com/edx/edx-platform/blob/master/openedx/core/lib/api/utils.py
```

Comment:
The link seems to be incorrect.

Reply:
Correct, this is pointing to the `edx:master` branch, so it will be valid once we merge this.

Reply:
Oh yes! Thanks for clarification. ))